packages/core/admin/admin/src/pages/MarketplacePage/tests/index.test.js

```diff
@@ -91,34 +72,21 @@ describe('Marketplace page - layout', () => {
     const trackUsage = jest.fn();
     useTracking.mockImplementationOnce(() => ({ trackUsage }));
 
-    const { container } = setup();
+    const { queryByText, getByRole } = render();
```

Comment:
I thought it was [recommended](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen) to use screen 🤔

Reply:
at the moment there's no certified direction nor eslint ruling – I think I spoke to @markkaylor about this and said I had no preference and I would be happy to see it brought up in an FE sync and followed through with the necessary work to implement a consistent direction 😄

Reply:
I tend to trust the article above since he's the author of the library, and it's quite nice not having to destructure all the queries in every test. But yes I'm happy bring it up in a FE sync 👍